patches/kaggle_gcp.py

```diff
@@ -52,6 +56,16 @@ def __init__(self, client):
         self._EXTRA_HEADERS["X-KAGGLE-PROXY-DATA"] = os.getenv(
             "KAGGLE_DATA_PROXY_TOKEN")
 
+    def api_request(self, *args, **kwargs):
+        """Wrap Connection.api_request in order to handle errors gracefully.
+        """
+        try:
+            super().api_request(*args, **kwargs)
+        except Forbidden as e:
+            print("Got a Permission Denied using Kaggle's Public Tier. "
```

Comment:
Proposed text: "Permission denied using Kaggle's public BigQuery integration. Did you mean to select a BigQuery account in the Kernels Settings sidebar?"